.github/workflows/release.yml

```diff
@@ -3,93 +3,14 @@ name: 🕊 Release
 on:
   release:
     types: [published]
-  schedule:
-    - cron: "0 7 * * *" # every day at 12AM PST
 
 jobs:
-  nightly:
-    name: 🌒 Nightly Release
-    if: github.repository == 'remix-run/remix' && github.event_name == 'schedule'
+  manual:
+    name: 📝 Manual Release
+    if: |
+      github.repository == 'remix-run/remix' &&
+      !contains(github.ref, 'nightly')
```

Comment:
actually! being that we use a dispatch event now i think we can use the standard GitHub action token when creating a release; please hold!

Reply:
nope, this workaround is needed to have the others run 😕

Reply:
Could you add a comment explaining that?